README.md

```diff
@@ -18,106 +23,143 @@ Installation
 ------------
 
 Add the following line to you `composer.json` file under `require`.
-```json
```

Comment:
@okaufmann it looks better to use the code hints, it helps GH to add syntax highlighting to the source codes. What do you think?